.github/workflows/close_old_prs.yaml

```
@@ -1,39 +1,34 @@
-name: Close Old PRs
+name: Close PRs Without Linked Issues
 on:
   schedule:
     - cron: "0 0 * * *"
 
 jobs:
-  close_old_prs:
+  close_prs_without_issue:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Check and Close Old PRs
+      - name: Check and Close PRs Without Linked Issues
         id: close_prs
         uses: actions/github-script@v6
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           script: |
             const daysThreshold = 45;
```

[Proofreading] I added a message component here, and I wonder if I placed it in the right spot @ujjawaltyagii. 
```suggestion
        with:
          script: |
            const daysThreshold = 45;
            comment: "This pull request has been closed because it does not mention the issue that it solves. Please refer to the [Contributing files](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#issues) to help you add this information. Then, tag the maintainers so your PR can be reopened."
```

---

@CBID2 I guess the placement isn't right, Ig may be we should use `github.issues.createComment` method inside the if statement to print this comment :
like this 
```
if (!linkedIssue) {
                  await github.pulls.update({ owner, repo, pull_number: pr.number, state: 'closed' });
                  await github.issues.createComment({
                    owner,
                    repo,
                    issue_number: pr.number,
                    body: "This pull request has been closed because it does not mention the issue that it solves. Please refer to the [Contributing files](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#issues) to help you add this information. Then, tag the maintainers so your PR can be reopened."
                  });
                }
              }
            });
```

---

Ok, add it in there @ujjawaltyagii.

---

done !!